src/sentry/notifications/helpers.py

```diff
@@ -702,21 +695,24 @@ def get_query(
         Q(
             (Q(user_id__in=user_ids) | Q(team_id__in=team_ids)),
             scope_type=NotificationScopeEnum.PROJECT.value,
-            scope_identifier=project.id,
+            scope_identifier__in=[project.id for project in projects],
         )
-        if project
+        if projects
         else Q()
     )
 
+    org_id = (
+        organization.id
+        if organization
+        else (projects[0].organization.id if projects else None)  # type:ignore
```

Comment:
```suggestion
        else (projects[0].organization_id if projects else None)  # type:ignore
```

We don't need to load the organization record to get its id.